File: solutions/scala/037-group-anagrams.scala
/**
 * Problem 37: Group Anagrams (LeetCode 49)
 * Difficulty: Med
 * Language: Scala
 */
def groupAnagrams(strs: Array[String]): List[List[String]] =
  strs.groupBy(_.sorted).values.map(_.toList).toList

File: solutions/elixir/037-group-anagrams.ex
# Problem 37: Group Anagrams (LeetCode 49)
# Difficulty: Med
# Language: Elixir
# 
defmodule GroupAnagrams do
  def group_anagrams(strs) do
    strs
    |> Enum.group_by(&(String.graphemes(&1) |> Enum.sort() |> Enum.join()))
    |> Map.values()
  end
end
